sgkit/io/vcf/vcf_reader.py

```diff
@@ -525,6 +525,10 @@ def vcf_to_zarr_sequential(
             ds.attrs["filters"] = filters
             ds.attrs["vcf_zarr_version"] = "0.1"
             ds.attrs["vcf_header"] = vcf.raw_header
+            try:
+                ds.attrs["contig_lengths"] = vcf.seqlens
+            except AttributeError:
```

Comment:
Checking the `cyvcf2` source I see that this raises if the lengths are absent in the header, so seem ok to then omit them in our output.